### PR TITLE
Add Docker command to README

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,13 @@ is the recommended way of installing Alertmanager.
 
 ### Docker images
 
-Docker images are available on [Quay.io](https://quay.io/repository/prometheus/alertmanager).
+Docker images are available on [Quay.io](https://quay.io/repository/prometheus/alertmanager) or [Docker Hub](https://hub.docker.com/r/prom/alertmanager/).
+
+You can launch an Alertmanager container for trying it out with
+
+    $ docker run --name alertmanager -d -p 127.0.0.1:9093:9093 prom/alertmanager
```
Comment (simonpasquier) [inline README.md:27]: Given the new rate limiting policies on Docker hub, can you change it to use quay.io/prometheus/alertmanager instead?

**Comment by simonpasquier** [comment]
You need to sign your last commit :)

**Comment by gliptak** [comment]
@simonpasquier yep there is no DCO in the browser. I will work this later tioday

**Comment by simonpasquier** [comment]
Thanks!

**Pull request merged**